### ISSUE TITLE
A11y: Search suggestions dialog is not closed when focus is lost

### ISSUE THREAD

**Issue by blacksheep--** [shopware/platform#7037]
*PHP Version:* 8.2   | *Shopware Version:* 6.6.9.0  | *Affected area / extension:* Platform(Default) 

 ----

*_Actual behaviour:_*

The search suggestions dialog box is not automatically closed when users continue to navigate, e.g. by tabbing. As soon as the keyboard focus leaves the search suggestions area of the main search field (CSS selector input.header-search-input), the keyboard focus moves behind the search suggestions dialog, especially on narrower viewports.

*_Expected behaviour:_*

The search suggestions dialog must be closed automatically after leaving the box or the input.

*_How to reproduce:_*

Enable `ACCESSIBILITY_TWEAKS` Feature-Flag

1. type into the search field until suggestions are showing
2. navigate away using tab

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **NEXT-40811**



[created from NEXT-40811, comment 541344]

**Comment by shopwareBot** [comment]
[public] 

 > Priority changed to: **High**

 Thank you for your report. Based on the information in this report, we have decided to investigate this issue in more detail and are planning to provide a fix for this issue. Although we are always eager to solve such problems as soon as we can, we can't give you a more specific time frame right now. We appreciate your patience and thank you for your understanding.



[created from NEXT-40811, comment 541561]
